wordpress-mobile/WordPress-iOS
# Nuking (Quite a few!!) Force Unwraps

### Details:

**ServiceRemoteWordPressComREST** lacked nullability specifiers for both, it's initializer, and it's public methods. As a result, we had a million (+1) force unwraps, and regular unwraps... all over, for an entity that doesn't really have failable initialization.

In this PR we're adding Nullability Specifiers (for the master of all REST classes), and doing a major cleanup all over.

### To test:
1. Verify that the unit tests don't break
2. Perform a fresh install
3. Run a smoke test: verify that you can log in, retrieve your blogs, and that the app looks good.

@bummytime sir, may i bug you with this one?
Thanks in advance!!
